src/core/system_impl.cpp

```diff
@@ -905,6 +905,24 @@ SystemImpl::make_command_flight_mode(FlightMode flight_mode, uint8_t component_i
         case FlightMode::OFFBOARD:
             custom_mode = px4::PX4_CUSTOM_MAIN_MODE_OFFBOARD;
             break;
+        case FlightMode::MANUAL:
```

Comment:
Those are not your changes and appear because your branch is not up-to-date with `develop` :-)